src/pages/Login/LoginPage.tsx

```diff
@@ -0,0 +1,169 @@
+      this.handleSubmit(e);
+    }
+  };
+
+  render() {
+    const { store } = this.context;
+    store.subscribe(() => {
+      const actualState = store.getState().authentication;
+      if (actualState.error !== undefined) {
+        this.setState({
+          submitted: false,
+          showError: true,
+          messageError: actualState.message
+        });
+      }
+      if (actualState.loggedIn) {
+        this.setState({
+          submitted: false,
+          showError: false
+        });
+      }
+    });
+    return (
+      <div className={'login-pf-page'}>
+        <div className={'container-fluid'}>
+          <Row>
+            <Col sm={8} smOffset={2} md={6} mdOffset={3} lg={6} lgOffset={3}>
+              <header className={'login-pf-page-header'}>
+                <img className={'login-pf-brand'} src={kialiTitle} alt={'logo'} />
+              </header>
+              <Row>
+                <Col sm={10} smOffset={1} md={8} mdOffset={2} lg={8} lgOffset={2}>
+                  <div className={'card-pf'}>
+                    <header className={'login-pf-header'}>{this.state.submitted && <h3>Logging...</h3>}</header>
+                    {this.state.showError && <Alert>{this.state.messageError}</Alert>}
+                    <Form onSubmit={e => this.handleSubmit(e)} id={'kiali-login'}>
+                      <FormGroup>
+                        <FormControl
+                          id="username"
+                          type="text"
+                          name="username"
+                          onChange={this.handleChange}
+                          placeholder={'Username'}
+                          disabled={false}
+                          required={true}
+                          onKeyPress={this.handleKeyPress}
+                        />
+                        {this.state.submitted && !this.state.username && <HelpBlock>Username is required</HelpBlock>}
+                      </FormGroup>
+                      <FormGroup>
+                        <FormControl
+                          type="password"
+                          name="password"
+                          onChange={this.handleChange}
+                          placeholder={'Password'}
+                          disabled={false}
+                          required={true}
+                          onKeyPress={this.handleKeyPress}
+                        />
+                        {this.state.submitted && !this.state.password && <HelpBlock>Password is required</HelpBlock>}
+                      </FormGroup>
+                      <Button
+                        type="submit"
+                        onKeyPress={this.handleKeyPress}
+                        className="btn btn-primary btn-block btn-lg"
+                      >
+                        Log In
+                      </Button>
+                    </Form>
+                  </div>
+                  <footer className={'login-pf-page-footer'}>
```

Comment:
Suggestion, not blocking (we can do it via a later PR): we could group all these "social links" into a dedicated component, that we would reuse in the about page too.
It could also include twitter and medium (we have a medium, don't we?)